test/integration/agent_policy_detailed_example/controls/gcloud.rb

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Google LLC
+# Copyright 2020 Google LLC
```

Comment:
Shouldn't there be some more tests here to verify the policy was actually created?

Reply:
Yup, added the extra validation